test/integration/helpers.bash

```diff
@@ -12,6 +12,19 @@ else
     ARCH="386"
 fi
 MACHINE_BIN_NAME=docker-machine_$PLATFORM-$ARCH
+BATS_LOG=${MACHINE_ROOT}/bats.log
+
+touch ${BATS_LOG}
```

Comment:
I don't have `truncate` on my Mac, so I've changed this for now - we can modify it later to something more elegant if we want.